Gruntfile.js

```diff
@@ -69,6 +70,18 @@ module.exports = exports = function(grunt) {
                         // just to add the BANNER
                         // without adding an extra grunt module
                         'dist/localforage.nopromises.js'
+                    ],
+                    // Node output files
+                    'dist/localforage.node.js': [
+                        'src/utils/node_banner.js',
+                        'bower_components/es6-promise/promise.js',
+                        'dist/localforage.nopromises.js'
+                    ],
+                    'dist/localforage.nopromises.node.js': [
+                        // just to add the BANNER
+                        // without adding an extra grunt module
+                        'src/utils/node_banner.js',
+                        'dist/localforage.nopromises.js'
```

Comment:
This is a bit of an unsatisfying pattern, because `dist/ `files (i.e. bundle files) are not typically used for Node dependencies.

Ideally people should be able to do:

```js
var localforage = require('localforage');
```

And this should "just work" in either the browser (via browserify/webpack/etc.) or Node. Typically the way to achieve that is with `"browser"` flags in `package.json`: e.g. [here's how PouchDB does it](https://github.com/pouchdb/pouchdb/blob/985f548f2cbd818ed0158ad8f97da31c79485654/package.json#L120-L128).

Also I'd say a `nopromises` version for Node is unnecessary, because large bundle sizes are less of an issue on the server side.